app/controllers/tariffs_controller.rb

```diff
@@ -1,44 +1,41 @@
 class TariffsController < ApplicationController
+  def bundle_tariff
```

Comment:
@romigos please move to **private** section

Reply:
fixed but don't know is it correct